engine/execution/computation/programs_test.go

```diff
@@ -64,7 +64,7 @@ func TestPrograms_TestContractUpdates(t *testing.T) {
 
 	// tx6 calls the method of the contract (version 2 expected)
 	tx6 := testutil.CreateEmitEventTransaction(account, account)
-	prepareTx(t, tx6, account, privKey, 4, chain)
+	prepareTx(t, tx6, account, privKey, 5, chain)
```

Comment:
the previous call in the test supposed to fail due to runtime error (failed in updating contract), previously we didn't increment the sequence number but with the new behaviour it needs to increment.